### DISCUSSION TITLE
Leaderboard on Taiko felt off

### DISCUSSION THREAD

**Discussion by rizkyrivaldi** [ppy/osu#24459] [Ideas] [answered]
The score leaderboard on taiko gameplay seems off. It overlaps with the drum thing so I'm unable to see whether the drum is hit by a key press if I have more than 8 scores in a map. For an improvement, I think it is better to move the leaderboard to the bottom part of the drum in this gamemode.

![image](https://github.com/ppy/osu/assets/81858136/904bb778-0e7f-4c1c-b971-61c4e2d9251b)

**Comment by Joehuu** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/20422
